### PR TITLE
[Docs] Fix the format of return

### PR DESCRIPTION
**Pull request by zhouzaida**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Fix the format of `return`

## Modification

mmcv/ops/*.py

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects, like MMDet or MMCls.
- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Review by ZwwWayne — APPROVED** [review]

File: mmcv/ops/group_points.py [hunk 1]
```diff
@@ -67,12 +67,14 @@ def __init__(self,
     def forward(self, points_xyz, center_xyz, features=None):
         """
         Args:
-            points_xyz (Tensor): (B, N, 3) xyz coordinates of the features.
```
Comment (ZCMax) [inline mmcv/ops/group_points.py:70]: xyz coordinates of the points.

File: mmcv/ops/group_points.py [hunk 2]
```diff
@@ -67,12 +67,14 @@ def __init__(self,
...
-            center_xyz (Tensor): (B, npoint, 3) coordinates of the centriods.
-            features (Tensor): (B, C, N) Descriptors of the features.
```
Comment (ZCMax) [inline mmcv/ops/group_points.py:72]: (B, C, N) The features of grouped points.

File: mmcv/ops/group_points.py [hunk 3]
```diff
@@ -67,12 +67,14 @@ def __init__(self,
...
+            points_xyz (torch.Tensor): (B, N, 3) xyz coordinates of the
+                features.
+            center_xyz (torch.Tensor): (B, npoint, 3) coordinates of the
+                centriods.
+            features (torch.Tensor): (B, C, N) Descriptors of the features.
 
         Returns:
-            Tensor: (B, 3 + C, npoint, sample_num) Grouped feature.
+            torch.Tensor: (B, 3 + C, npoint, sample_num) Grouped feature.
```
Comment (ZCMax) [inline mmcv/ops/group_points.py:77]: Grouped concatenated coordinates and features of points.

File: mmcv/ops/nms.py [hunk 4]
```diff
@@ -131,8 +131,8 @@ def nms(boxes, scores, iou_threshold, offset=0, score_threshold=0, max_num=-1):
         max_num (int): maximum number of boxes after NMS.
 
     Returns:
-        tuple: kept dets(boxes and scores) and indice, which is always the \
-            same data type as the input.
+        tuple: kept dets (boxes and scores) and indice, which is always the
+        same data type as the input.
```
Comment (ZCMax) [inline mmcv/ops/nms.py:135]: which is always the same data type -> which always have the same data type

File: mmcv/ops/nms.py [hunk 5]
```diff
@@ -200,8 +200,8 @@ def soft_nms(boxes,
         offset (int, 0 or 1): boxes' width or height is (x2 - x1 + offset).
 
     Returns:
-        tuple: kept dets(boxes and scores) and indice, which is always the \
-            same data type as the input.
+        tuple: kept dets (boxes and scores) and indice, which is always the
+        same data type as the input.
```
Comment (ZCMax) [inline mmcv/ops/nms.py:204]: is -> have

File: mmcv/ops/pixel_group.py [hunk 6]
```diff
@@ -13,21 +13,22 @@ def pixel_group(score, mask, embedding, kernel_label, kernel_contour,
...
-        kernel_contour (np.array or Tensor): The kernel contour with size hxw.
         kernel_region_num (int): The instance kernel region number.
         distance_threshold (float): The embedding distance threshold between
             kernel and pixel in one instance.
 
     Returns:
-        pixel_assignment (List[List[float]]): The instance coordinate list.
-            Each element consists of averaged confidence, pixel number, and
-            coordinates (x_i, y_i for all pixels) in order.
+        list[list[float]]: The instance coordinate list. Each element consists
```
Comment (ZCMax) [inline mmcv/ops/pixel_group.py:29]: coordinates

File: mmcv/ops/point_sample.py [hunk 7]
```diff
@@ -221,15 +227,15 @@ def rel_roi_point_to_rel_img_point(rois,
...
     Args:
-        rois (Tensor): RoIs or BBoxes, shape (N, 4) or (N, 5)
-        rel_roi_points (Tensor): Point coordinates inside RoI, relative to
-            RoI, location, range (0, 1), shape (N, P, 2)
-        img (tuple/Tensor): (height, width) of image or feature map.
+        rois (torch.Tensor): RoIs or BBoxes, shape (N, 4) or (N, 5)
+        rel_roi_points (torch.Tensor): Point coordinates inside RoI, relative
+            to RoI, location, range (0, 1), shape (N, P, 2)
+        img (tuple or torch.Tensor): (height, width) of image or feature map.
         spatial_scale (float): Scale points by this factor. Default: 1.
```
Comment (ZCMax) [inline mmcv/ops/point_sample.py:234]: (float, optional)

File: mmcv/ops/point_sample.py [hunk 8]
```diff
@@ -188,14 +194,14 @@ def abs_img_point_to_rel_img_point(abs_img_points, img, spatial_scale=1.):
     coordinates for sampling.
 
     Args:
-        abs_img_points (Tensor): Image based absolute point coordinates,
+        abs_img_points (torch.Tensor): Image based absolute point coordinates,
             shape (N, P, 2)
-        img (tuple/Tensor): (height, width) of image or feature map.
+        img (tuple or torch.Tensor): (height, width) of image or feature map.
         spatial_scale (float): Scale points by this factor. Default: 1.
```
Comment (ZCMax) [inline mmcv/ops/point_sample.py:200]: (float, optional)

File: mmcv/ops/point_sample.py [hunk 9]
```diff
@@ -245,14 +251,15 @@ def point_sample(input, points, align_corners=False, **kwargs):
...
 
     Args:
-        input (Tensor): Feature map, shape (N, C, H, W).
-        points (Tensor): Image based absolute point coordinates (normalized),
-            range [0, 1] x [0, 1], shape (N, P, 2) or (N, Hgrid, Wgrid, 2).
+        input (torch.Tensor): Feature map, shape (N, C, H, W).
+        points (torch.Tensor): Image based absolute point coordinates
+            (normalized), range [0, 1] x [0, 1], shape (N, P, 2) or
+            (N, Hgrid, Wgrid, 2).
         align_corners (bool): Whether align_corners. Default: False
```
Comment (ZCMax) [inline mmcv/ops/point_sample.py:258]: (bool, optional)

File: mmcv/ops/points_in_boxes.py [hunk 10]
```diff
@@ -18,7 +18,8 @@ def points_in_boxes_part(points, boxes):
             LiDAR/DEPTH coordinate, (x, y, z) is the bottom center
 
     Returns:
-        box_idxs_of_pts (torch.Tensor): (B, M), default background = -1
+        torch.Tensor: Return the box indies of points with the shape of
+        (B, M, T). Default background = 0.
```
Comment (ZCMax) [inline mmcv/ops/points_in_boxes.py:22]: Return the box indies of points with the shape of (B, M). Default background = -1.

File: mmcv/ops/points_sampler.py [hunk 11]
```diff
@@ -12,13 +12,13 @@ def calc_square_dist(point_feat_a, point_feat_b, norm=True):
...
-        point_feat_b (Tensor): (B, M, C) Feature vector of each point.
-        norm (Bool, optional): Whether to normalize the distance.
+        point_feat_a (torch.Tensor): (B, N, C) Feature vector of each point.
+        point_feat_b (torch.Tensor): (B, M, C) Feature vector of each point.
+        norm (bool, optional): Whether to normalize the distance.
             Default: True.
 
     Returns:
-        Tensor: (B, N, M) Distance between each pair points.
+        torch.Tensor: (B, N, M) Distance between each pair points.
```
Comment (ZCMax) [inline mmcv/ops/points_sampler.py:21]: Square distance between each point pair.

File: mmcv/ops/points_sampler.py [hunk 12]
```diff
@@ -92,11 +92,12 @@ def __init__(self,
     def forward(self, points_xyz, features):
         """
         Args:
-            points_xyz (Tensor): (B, N, 3) xyz coordinates of the features.
-            features (Tensor): (B, C, N) Descriptors of the features.
+            points_xyz (torch.Tensor): (B, N, 3) xyz coordinates of the
+                features.
+            features (torch.Tensor): (B, C, N) Descriptors of the features.
```
Comment (ZCMax) [inline mmcv/ops/points_sampler.py:97]: points_xyz (torch.Tensor): (B, N, 3) xyz coordinates of the points.
 features (torch.Tensor): (B, C, N) features of the points.

File: mmcv/ops/scatter_points.py [hunk 13]
```diff
@@ -25,10 +25,11 @@ def forward(ctx, feats, coors, reduce_type='max'):
...
         Returns:
-            voxel_feats (torch.Tensor): [M, C]. Reduced features, input
-                features that shares the same voxel coordinates are reduced to
-                one row.
-            voxel_coors (torch.Tensor): [M, ndim]. Voxel coordinates.
+            tuple[torch.Tensor]: A tuple contains two elements. The first one
+            is the reduced features whose shape is [M, C] and which are reduced
+            from input features that share the same voxel coordinates are
+            reduced to one row. The second is voxel coordinates whose shape is
+            [M, ndim].
```
Comment (ZCMax) [inline mmcv/ops/scatter_points.py:32]: tuple[torch.Tensor]: A tuple contains two elements. The first one is the voxel features with shape [M, C] which are respectively reduced from input features that share the same voxel coordinates . The second is voxel coordinates with shape [M, ndim].

File: mmcv/ops/scatter_points.py [hunk 14]
```diff
@@ -88,9 +89,10 @@ def forward_single(self, points, coors):
                 multi-dim voxel index) of each points.
 
         Returns:
-            voxel_feats (torch.Tensor): Reduced features, input features that
-                shares the same voxel coordinates are reduced to one row.
-            voxel_coors (torch.Tensor): Voxel coordinates.
+            tuple[torch.Tensor]: A tuple contains two elements. The first one
```
Comment (ZCMax) [inline mmcv/ops/scatter_points.py:92]: same as above

File: mmcv/ops/scatter_points.py [hunk 15]
```diff
@@ -104,9 +106,10 @@ def forward(self, points, coors):
...
```
Comment (ZCMax) [inline mmcv/ops/scatter_points.py:109]: same

File: mmcv/ops/three_interpolate.py [hunk 16]
```diff
@@ -21,14 +21,14 @@ def forward(ctx, features: torch.Tensor, indices: torch.Tensor,
                 weight: torch.Tensor) -> torch.Tensor:
         """
         Args:
-            features (Tensor): (B, C, M) Features descriptors to be
+            features (torch.Tensor): (B, C, M) Features descriptors to be
                 interpolated
-            indices (Tensor): (B, n, 3) index three nearest neighbors
+            indices (torch.Tensor): (B, n, 3) index three nearest neighbors
                 of the target features in features
```
Comment (ZCMax) [inline mmcv/ops/three_interpolate.py:27]: indices of three nearest neighbor features for the target features.

File: mmcv/ops/three_interpolate.py [hunk 17]
```diff
@@ -21,14 +21,14 @@ def forward(ctx, features: torch.Tensor, indices: torch.Tensor,
...
-            weight (Tensor): (B, n, 3) weights of interpolation
+            weight (torch.Tensor): (B, n, 3) weights of interpolation
```
Comment (ZCMax) [inline mmcv/ops/three_interpolate.py:28]: weights of three nearest neighbor features for the target features.

File: mmcv/ops/voxelize.py [hunk 18]
```diff
@@ -36,13 +36,12 @@ def forward(ctx,
...
-            coors_out (torch.Tensor): Output coordinates with the shape of
-                [M, 3].
-            num_points_per_voxel_out (torch.Tensor): Num points per voxel with
-                the shape of [M]. Only returned when max_points != -1.
+            tuple[torch.Tensor]: A tuple contains three elements. The first one
+            is the voxelization whose shape is [M, max_points, ndim]. The
+            voxelization only contain points and returned when
+            max_points != -1. The second is the coordinates with the shape of
+            [M, 3]. The last is the number of points per voxelization with the
+            shape of [M]. Only returned when max_points != -1.
```
Comment (ZCMax) [inline mmcv/ops/voxelize.py:44]:
```suggestion
            tuple[torch.Tensor]: A tuple contains three elements. The first one
            is the output voxels with the shape of [M, max_points, n_dim], 
            which only contain points and returned whe max_points != -1. 
            The second is the voxel coordinates with shape of [M, 3]. 
            The last is number of point per voxel with the shape of [M], which 
            only returned when max_points != -1.
```

File: mmcv/ops/three_nn.py [hunk 19]
```diff
@@ -20,14 +20,14 @@ def forward(ctx, target: torch.Tensor,
...
                 find the nearest neighbors.
-            source (Tensor): shape (B, M, 3), points set that is used
+            source (torch.Tensor): shape (B, M, 3), points set that is used
                 to find the nearest neighbors of points in target set.
 
         Returns:
-            Tensor: shape (B, N, 3), L2 distance of each point in target
-                set to their corresponding nearest neighbors.
+            torch.Tensor: shape (B, N, 3), L2 distance of each point in target
+            set to their corresponding nearest neighbors.
```
Comment (ZCMax) [inline mmcv/ops/three_nn.py:30]:
```suggestion
            set to their corresponding top-3 nearest neighbors.
```

File: mmcv/ops/points_in_boxes.py [hunk 20]
```diff
@@ -67,7 +68,8 @@ def points_in_boxes_cpu(points, boxes):
             (x, y, z) is the bottom center.
 
     Returns:
-        box_idxs_of_pts (torch.Tensor): (B, M, T), default background = 0.
+        torch.Tensor: Return the box indies of points with the shape of
```
Comment (ZCMax) [inline mmcv/ops/points_in_boxes.py:71]:
```suggestion
        torch.Tensor: Return the box indices of points with the shape of
```

File: mmcv/ops/points_in_boxes.py [hunk 21]
```diff
@@ -102,7 +104,8 @@ def points_in_boxes_all(points, boxes):
...
```
Comment (ZCMax) [inline mmcv/ops/points_in_boxes.py:107]:
```suggestion
        torch.Tensor: Return the box indices of points with the shape of
```

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**